packages/stacked_services/lib/src/navigation_service.dart

```diff
@@ -96,45 +96,80 @@ class NavigationService {
   /// - rightToLeftWithFade
   /// - leftToRightWithFade
   /// - cupertino
-  Future<dynamic>? navigateWithTransition(Widget page,
-      {bool? opaque,
-      String transition = '',
-      Duration? duration,
-      bool? popGesture,
-      int? id}) {
-    return Get.to(
+  Future<T?>? navigateWithTransition<T>(
```

Comment:
If you make this `T` then you can pass in a nullable type or non-nullable type. If you make it `T?` it always has to be a nullable type. We should change it to `T`

Reply:
@FilledStacks  It's the return type for `T` but not the actual argument, the actual generic type argument is here `navigateWithTransition<T>` so it doesn't have to be always of nullable type